docs/developer/example-configurations/index.rst

```diff
@@ -6,5 +6,4 @@ Example configurations
 .. toctree::
    :maxdepth: 2
 
-   Developing against gitlab.com <gitlab>
-   Developing against another renku deployment <renkulab>
+   Deploying Renku with an external GitLab <gitlab>
```

Comment:
yeah - I was thinking the same. Actually, describing how to deploy against another gitlab should probably be in the admin docs and the developer docs should only contain the minikube-specific aspects of it?

Reply:
I created an [issue](https://github.com/SwissDataScienceCenter/renku/issues/1109) for this as it would require restructuring things quite a bit more and I don't want to tackle this right now.